parquet-hadoop/src/main/java/org/apache/parquet/hadoop/util/SerializationUtil.java

```diff
@@ -53,22 +53,14 @@ private SerializationUtil() { }
    * @throws IOException if there is an error while writing
    */
   public static void writeObjectToConfAsBase64(String key, Object obj, Configuration conf) throws IOException {
-    ByteArrayOutputStream baos = null;
-    GZIPOutputStream gos = null;
-    ObjectOutputStream oos = null;
-
-    try {
-      baos = new ByteArrayOutputStream();
-      gos = new GZIPOutputStream(baos);
-      oos = new ObjectOutputStream(gos);
-      oos.writeObject(obj);
-    } finally {
-      Closeables.close(oos);
-      Closeables.close(gos);
-      Closeables.close(baos);
+    try(ByteArrayOutputStream baos = new ByteArrayOutputStream()) {
```

Comment:
You can consolidate these into a single `try` e.g.:

```
try (ByteArrayOutputStream baos = new ByteArrayOutputStream();
       GZIPOutputStream gos = new GZIPOutputStream(baos);
       ObjectOutputStream oos = new ObjectOutputStream(gos)) {
    oos.writeObject(obj);
    conf.set(key, new String(Base64.encodeBase64(baos.toByteArray()), "UTF-8"));
}
```

Reply:
Sweet suggestion, thanks!